src/Symfony/Bridge/ProxyManager/Tests/LazyProxy/PhpDumper/ProxyDumperTest.php

```diff
@@ -80,6 +80,37 @@ public function testGetProxyFactoryCode()
         );
     }
 
+    /**
+     * @dataProvider getPrivatePublicDefinitions
+     */
+    public function testCorrectPublicPrivateInstantiation(Definition $definition, string $access)
```

Comment:
I hate to be picky, but what doesn it mean "correct" in this context?

Reply:
Changed name